Makefile

```diff
@@ -110,11 +139,12 @@ dev-install-dependencies-lib:
 dev-install-dependencies-for-development:
 	pip install --upgrade-strategy only-if-needed -r requirements-dev.txt
 
-e2e:
-	npx playwright test
+e2e-install:
+	npm ci
 
-e2e-headed:
+e2e-test:
+	npx playwright test
```

Comment:
Usually, these commands are defined as `scripts` in `package.json` (eg. [here](https://github.com/scalyr/scalyr-webapp/blob/master/packages/e2e/playwright/package.json#L13)) so it's all defined in one place and it's easy to see what scripts are available.

Scripts in `package.json` then can run as `npm run xxx`.

Reply:
Fixed